providers/tailscale/resources/tailscale.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package resources
+
+import (
+	"context"
+
+	"go.mondoo.com/cnquery/v11/providers/tailscale/connection"
+)
+
+func (r *mqlTailscale) id() (string, error) {
+	// TODO need to set the tailnet
```

Comment:
I think it is a good idea to implement this in this PR.

Reply:
will do 🥇